### DISCUSSION TITLE
Getting the "Galaxy Brain" badge doesn't work?

### DISCUSSION THREAD

**Discussion by epistax1s** [community/community#18550] [Other Feature Feedback, Questions, & Ideas] [answered]
I see a lot of comments from people who fulfilled all the conditions, but did not receive the badge.

**Comment by isinicolle** [comment]
A veces la vida no nos sonríe a todos :(

**Comment by AnujMutha** [comment]
hey @bogomolovNikita,
There is a single condition you need to satisfy, "Answer 2 discussions (Q&A Cateogary disscussion)"
Voila, you have a badge on your profile :)

**Comment by aphiratnimanussonkul** [comment]
you need to answer 2 discussions

**Reply by saivarshithm** [reply]
ok

**Reply by ramazanix** [reply]
Ok

**Comment by vityakut** [comment]
Answer two discussions

**Comment by fayasnoushad** [comment]
I answered 4 discussions, but did not receive the badge for me.

**Reply by AnujMutha** [reply]
did they approved your answer ?

**Reply by AnujMutha** [reply]
It is essential that they must approve your answer to qualify for a badge

**Reply by fayasnoushad** [reply]
New discussion :- #18571

**Comment by ltthuong** [comment]
Get 2 accepted answers

**Comment by mathishouis** [comment]
Your two answers need to be approved

**Comment by Chathu12345** [comment]
> I see a lot of comments from people who fulfilled all the conditions, but did not receive the badge.

When you answer two questions in a discussions you can unlock the badge. If some user marked your answers then you can have that badge. But you have to have two answers as marked answers.

**Comment by queenofcorgis** [comment, marked as answer]
Hi there 👋🏼  - for answers to qualify for the Galaxy Brain 🧠  achievement, the answers must be marked as such by another user, not the original poster!

**Comment by nusgnojkrap** [comment]
Just answer two discussions to get one

**Comment by kereh** [comment]
answering two discussions
